platform/docs/docs/platform/services/customization-service/customizationService.md

```diff
@@ -0,0 +1,467 @@
+---
+sidebar_label: Introduction
+sidebar_position: 1
+---
+
+import { customizations, TableGenerator } from './sampleCustomizations';
+import Heading from '@theme/Heading';
+import TOCInline from '@theme/TOCInline';
+
+# Customization Service
+
+There are a lot of places where users may want to configure certain elements
+differently between different modes or for different deployments.  A mode
+example might be the use of a custom overlay showing mode related DICOM header
+information such as radiation dose or patient age.
+
+The use of `customizationService` enables these to be defined in a typed fashion by
+providing an easy way to set default values for this, but to allow a
+non-default value to be specified by the configuration or mode.
+
+
+:::note
+
+`customizationService` itself doesn't implement the actual customization,
+but rather just provide mechanism to register reusable prototypes, to configure
+those prototypes with actual configurations, and to use the configured objects
+(components, data, whatever).
+
+Actual implementation of the customization is totally up to the component that
+supports customization.
+:::
+
+
+## General Overview
+
+This framework allows you to configure many features, or "slots," through customization modules. Extensions can choose to offer their own module, which outlines which values can be changed. By looking at each extension's getCustomizationModule(), you can see which objects or components are open to customization.
+
+Below is a high-level example of how you might define a default customization and then consume and override it:
+
+1. **Defining a Customizable Default**
+
+   In your extension, you might export a set of default configurations (for instance, a list that appears in a panel). Here, you provide an identifier and store the default list under that identifier. This makes the item discoverable by the customization service:
+
+   ```js
+   // Inside your extension’s customization module
+   export default function getCustomizationModule() {
+     return [
+       {
+         name: 'default',
+         value: {
+           defaultList: ['Item A', 'Item B'],
+         },
+       },
+     ];
+   }
+   ```
+
+   By naming it `default`, it is automatically registered.
```

Comment:
Also explain that by using other names, that set of customizations can be registered as an entire set by refererring to the entire name - that is the point of the register commands, so that something like:
{
  name: 'myCustomizationSet',
  value: {
    ... customizations here
  }
}
could be applied by registering the customizations ['myCustomizationSet']
That is also intended to allow nested named sets to be registered, eg by referring to another set within that set.